builds/e2e/nested-longhaul.yaml

```diff
@@ -1,119 +1,64 @@
 trigger: none
 pr: none
 
-variables:
-  images.artifact.name.linux: 'core-linux'
-  vsts.project: $(System.TeamProjectId)
-  # Variable defined in VSTS
-  #azure.keyVault: ''
-  #azure.subscription: ''
-  #edgelet.package.build: <edglet package build id>
-  #images.build: <images build id>
-  #pool.name: ''
-  #container.registry: ''
-  #snitch.storage.account: ''
-  # Variables settable at queue time
-  #test.startDelay.arm
-  #test.startDelay.amd
-  #edgelet.package.branchName
-  #images.branchName
-  #loadGen.message.frequency.amd64: 'xx:xx:xx.xx'
-  #loadGen.message.frequency.arm32v7: 'xx:xx:xx.xx'
-  #loadGen.message.frequency.arm64v8: 'xx:xx:xx.xx'
-  #run.linux.amd64.moby: 'true/false'
-  #run.linux.arm32v7.moby: 'true/false'
-  #run.linux.arm64v8.docker: 'true/false'
-  #longhaul.desiredModulesToRestartCSV: ''
-  #longhaul.restartIntervalInMins: ''
-  #twinTester.twinUpdateSize: ''
-  #twinTester.twinUpdateFrequency: 'xx:xx:xx.xx' 
-  #twinTester.twinUpdateFailureThreshold: 'xx:xx:xx.xx'
-  #twinTester.edgeHubRestartFailureTolerance: 'xx:xx:xx.xx'
-  #metricsCollector.metricsEndpointsCSV
-  #metricsCollector.scrapeFrequencyInSecs: 'xx:xx:xx.xx'
-  #metricsCollector.uploadTarget
+resources:
+  pipelines:
+  - pipeline: images
+    source: 'Azure-IoT-Edge-Core Build Images'
+    branch: master
+  - pipeline: packages
+    source: 'Azure-IoT-Edge-Core Edgelet Packages'
+    branch: master
 
-jobs:
-################################################################################
-  - job: linux_amd64_moby
-################################################################################
-    displayName: Linux AMD64 Moby
-    condition: ne(variables['agent.group'], '')
+stages:
+- stage: SetupVMs
+  jobs:
+  - template: templates/nested-parent-vm-setup.yaml
+  - job:  Deploy_Longhaul_Linux_Amd64
+    displayName: Set up and run longhaul tests on Linux Amd64
+    dependsOn: SetupVM_level4
+    condition: and(succeeded(), eq(variables['run.linux.amd64.moby'], 'true'))
     pool:
       name: $(pool.name)
       demands:
         - agent-group -equals $(agent.group)
         - Agent.OS -equals Linux
         - Agent.OSArchitecture -equals X64
-        - run-long-haul -equals true
-        - nested-edge -equals L3
+        - run-idle -equals true 
```

Comment:
Why are you changing `run-long-haul` to `run-idle`?
Please use a unique capability to indicate the test agents are used for nested long haul. 
Maybe `run-nested-long-haul`

Reply:
Please see other comments about `agent-group` and question about `run-idle`.